src/cljs/kixi/hecuba/tabs/hierarchy/tapestry.cljs

```diff
@@ -0,0 +1,385 @@
+(ns kixi.hecuba.tabs.hierarchy.tapestry
+  (:require-macros [cljs.core.async.macros :refer [go go-loop]]
+                   [hiccups.core :as hiccups])
+  (:require [clojure.string :as str]
```

Comment:
This does weird things to the require-macros key, so perhaps not 100% CLJS proof.